libs/langchain/langchain/document_loaders/s3_directory.py

```diff
@@ -8,7 +8,7 @@
 class S3DirectoryLoader(BaseLoader):
     """Load from `Amazon AWS S3` directory."""
 
-    def __init__(self, bucket: str, prefix: str = ""):
+    def __init__(self, bucket: str, prefix: str = "", **kwargs: Any):
```

Comment:
slight preference to just pass these in as boto_kwargs: Dict

Reply:
ah yea i meant drop the `**` as well, think that's better reserved for things like potentially passing through kwargs to parent classes (in case that were ever to happen)

Reply:
In that case I would rather add explicit arguments. There are not so many of them.

Reply:
Just pushed a new proposal. This time with explicit arguments.
I think it's nicer as you get hints, docs, IDE auto-completion, ...

Reply:
Hi @baskaryan , if you have time, could you have a look again ?